sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/tests/Scenario/StorageClassTests.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.ContainerOrchestratorRuntime.Models;
+using Azure.ResourceManager.Kubernetes;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.ContainerOrchestratorRuntime.Tests.Tests
+{
+    [TestFixture]
+    public class StorageClassTests : ContainerOrchestratorRuntimeManagementTestBase
+    {
+        public StorageClassTests() : base(true)
```

Comment:
Any reason you run this test in async-only mode but run `LoadBalancerTests` in sync-only mode? Normally we can just have a parameter `bool isAnsyc` in the test constructor, and the test framework will run both sync and async for you. See https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core.TestFramework#sync-async-tests

Reply:
Changed `LoadBalancerTests` to async

Reply:
The tests keep failing with message 

```
Setup failed for test fixture Azure.ResourceManager.ContainerOrchestratorRuntime.Tests.Tests.StorageClassTests
No suitable constructor was found
  Failed CreateStorageClassAsync() [1 ms]
  Error Message:
   OneTimeSetUp: No suitable constructor was found
```

Thus we are testing only async methods for now.